schema/job/compute.json

```diff
@@ -16,15 +16,29 @@
         },
         "nodes": {
             "description": "number of nodes used for the job inside the RMS.",
-            "type": "integer"
+            "type": "integer",
+            "validateNodes": {},
+            "errorMessage": "Max node count for selected queue exceeded"
         },
         "ppn": {
             "description": "number of CPUs used for the job inside the RMS.",
-            "type": "integer"
+            "type": "integer",
+            "validatePpn": {},
+            "errorMessage": "Max count exceeded"
         },
         "timeLimit": {
             "description": "Wallclock time limit for computing a job. Clock format: 'hh:mm:ss'",
-            "type": "string"
+            "type": "string",
+            "validateTimeLimit": {},
+            "errorMessage": "Time, 00:00:00 - 99:59:59"
```

Comment:
Do we only allow for wall time limits below 100h? Some calculations may take longer than 4 days.

Reply:
This is a pre-existing validation logic but yes I agree.